QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,135 @@
+This document is a declaration of software quality for the `class_loader` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `class_loader` Quality Declaration
+
+The package `class_loader` claims to be in the **Quality Level 1** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 1 in REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+## Version Policy [1]
+
+### Version Scheme [1.i]
+
+`class_loader` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#versioning), and is at or above a stable version. **The current version is 1.4.0**.
+
+### Version Stability [1.ii]
+
+### Public API Declaration [1.iii]
+
+All symbols in the installed headers are considered part of the public API.
+
+All installed headers are in the `include` directory of the package, headers in any other folders are not installed and considered private.
+
+### API Stability Within a Released ROS Distribution [1.iv]/[1.vi]
+
+`class_loader` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released.
+
+### ABI Stability Within a Released ROS Distribution [1.v]/[1.vi]
+
+`class_loader` will maintain ABI stability within a ROS distribution.
+
+## Change Control Process [2]
+
+`class_loader` follows the recommended guidelines for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#change_control_process).
+
+### Change Requests [2.i]
+
+All changes occur through a pull request
+
+### Contributor Origin [2.ii]
+
+Must have confirmation of contributor origin (e.g. DCO, CLA, etc.).
+
+### Peer Review Policy [2.iii]
+
+All pull request have two peer reviews
+
+### Continuous Integration [2.iv]
+
+all pull request must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers)
+
+### Documentation Policy [2.v]
+
+All pull request must resolve related documentation changes before merging
+
+## Documentation [3]
+
+### Feature Documentation [3.i]
+
+`class_loader` has a [feature list](http://wiki.ros.org/class_loader) and each item in the list links to the corresponding feature documentation.
+There is documentation for all of the features, and new features require documentation before being added.
+
+### Public API Documentation [3.ii]
+
+`class_loader` has embedded API documentation and it is generated using doxygen and is [hosted](http://wiki.ros.org/class_loader) along side the feature documentation.
```

Comment:
I don't think this should link to ros1 docs, and ROS 2 API docs aren't currently being generated for class_loader

https://github.com/ros2/docs.ros2.org/blob/a5ed03490741da161598f3b639d47eb8f361984e/Makefile#L4-L14